cucumber-rails.gemspec

```diff
@@ -23,11 +23,8 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency('capybara', ['>= 2.18', '< 4'])
   s.add_runtime_dependency('cucumber', '>= 3.2', '< 9')
-  s.add_runtime_dependency('mime-types', ['~> 3.3'])
   s.add_runtime_dependency('nokogiri', '~> 1.10')
   s.add_runtime_dependency('railties', ['>= 5.0', '< 8'])
-  s.add_runtime_dependency('rexml', '~> 3.0') # rexml is a bundled gem from ruby 3
```

Comment:
We still support lower ruby versions. I'm kinda on the fence about this, as ideally people should be controlling their own version constraints if they need this. So I think I'm 50-50 about this. I don't see the harm in retaining this until we're ruby 3.0+ only (Which isn't likely to be too long at current development rates).

That being said, if you're passionate about this, I don't see the harm in making this change. It was originally made when we introduced ruby3 into our support matrix.

Reply:
My argument is there is no code in the `cucumber-rails` gem that uses `rexml` or `webrick`. It operates identically, with or without these dependencies, on all supported versions of Ruby. There is no value in declaring them as runtime dependencies.